.github/workflows/release.yml

```diff
@@ -3,6 +3,7 @@ on:
   push:
     branches:
       - main
+      - next
```

Comment:
in the `release` step we check again if it's main
you need to change that one as well and maybe you can also try with `dry_run`

Reply:
Done